build.py

```diff
@@ -72,6 +73,14 @@ def main(*argv):
         type=str,
         help="Python version to use in install"
     )
+    parser.add_argument(
+        "--hash",
+        type=str,
+        nargs="*",
+        default=[],
+        dest="hash_names",
+        help="Hashes to run on each installer"
+    )
```

Comment:
Allow hash(es) to be user specified and optional. Default is not to include one as before.